Lena/hw_1.1_table/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Формула удовольствия</title>
+  </head>
+  <body>
+    <table border="4" width="100%">
+      <tr>
+      <th align="center" colspan="2"><h1>Формула удовольствия</h1>
+      </th>
+    </tr>
+      <tr>
+        <td align="left">
+          <ul>
+            <a href="about.html">Функции удовольствия</a>
+          </ul>
+          <ul>
+            <a href="home.html">Что нам мешает
+              <br>получать удовольствие</a>
+          </ul>
+          <ul>
+            <a href="article.html">Топ 5 сожалений
+              <br>перед смертью</a>
+          </ul>
+        </td>
+        <td>
+        <ol>
+          <h4>Формула счастья достаточно проста и представлена 5-ю состовляющими:</h4>
```

Comment:
Forgot about < li> under < ol>

under < ol> and < ul> we can set only tag < li>, and under < li> we set all we need